pkg/installer/install

```diff
@@ -24,7 +24,7 @@
 #   eve_install_skip_rootfs
 #   eve_install_skip_zfs_checks
 #   eve_install_zfs_with_raid_level
-#   eve_install_kubevirt_etcd_sizeGB
+#   eve_install_k3s_etcd_sizeGB
```

Comment:
@eriknordmark , you have renamed `kubevirt` in some config properties to `kubernetes`, why is `k3s` being used here? Isn't better to just use `kubernetes` as well?

Reply:
@rene the naming suggestions were from @andrewd-zededa 
But we can put kubernetes here if that's preferred.